plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_launcher_util.py

```diff
@@ -57,6 +60,7 @@ def launch_job_on_ray(
         run_job_ray = ray.remote(_run_job)
 
     ret = run_job_ray.remote(
+        hydra_context=hydra_context,
```

Comment:
This one is different because the function did not take config_loader before.
Can you explain why it needs the hydra_context now?